aten/src/ATen/cpu/vec/vec_base.h

```diff
@@ -42,6 +42,15 @@
 #define __FORCE_INLINE __forceinline
 #endif
 
+#if defined(_MSC_FULL_VER)
+/*
+https://learn.microsoft.com/en-us/cpp/overview/compiler-versions?view=msvc-170
+Use _MSC_FULL_VER to identify current compiler is msvc,
+Windows llvm will not have this defination.
+*/
+#define __CUR_MSVC_CL
```

Comment:
why not using `_MSC_FULL_VER` directly?

Reply:
It is good to extend to more compilers, such as icx.

Reply:
name to __msvc_cl__